requirements/development.in

```diff
@@ -21,7 +21,7 @@
 ipython
 progress>=1.5,<2
 pyinstrument>=4.0.2,<5
-pylint==2.9.6
+pylint==2.13.9
```

Comment:
Is there a reason we're pinning `pylint` as opposed to leaving this unbounded and running `pip-compile-multi --upgrade-package=pylint`?

Reply:
If you download the latest version of pylint you have hundreds warnings